Documentation/deploy-master-single.md

```diff
@@ -112,7 +112,7 @@ spec:
     - --service-cluster-ip-range=${SERVICE_IP_RANGE}
     - --secure_port=443
     - --advertise-address=${ADVERTISE_IP}
-    - --admission-control=NamespaceLifecycle,NamespaceExists,LimitRanger,SecurityContextDeny,ServiceAccount,ResourceQuota
+    - --admission-control=NamespaceLifecycle,LimitRanger,SecurityContextDeny,ServiceAccount,ResourceQuota
```

Comment:
Sorry, missed the slack discussion. But I just tested and it looks like it's failing on the creation of the mirror pods. We've just been lucky with timing in automated deployments (create the namespace before the mirror pods are being created).

So if we leave namespace as is, the instruction order might be: setup master, setup kubectl, create namespace, create worker. Or change it back to default.